### PR TITLE
Modified CaseManagement to be asynchronous in the creation of an erra…

### PR DESCRIPTION
**Pull request by lsjolinder**
…nd in integrated systems.

### CODE REVIEW

File: src/test/java/se/sundsvall/casemanagement/TestUtil.java [hunk 1]
```diff
@@ -121,12 +122,24 @@ public static void mockFbPropertyOwners(FbClient fbClientMock, List<StakeholderD
         agareInfoResponse.setData(agareInfoDataItemList);
         Mockito.doReturn(agareInfoResponse).when(fbClientMock).getPropertyOwnerInfoByUuid(any(), any(), any(), any());
     }
+    
+    @Test
+    void doshit() throws JsonProcessingException {
```
Comment (mha109) [inline src/test/java/se/sundsvall/casemanagement/TestUtil.java:127]: bästa namnet =D

File: src/main/java/se/sundsvall/casemanagement/integration/db/model/CaseEntity.java [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+@ToString
+@Builder(setterPrefix = "with")
+@NoArgsConstructor(access = lombok.AccessLevel.PROTECTED)
+@AllArgsConstructor
+public class CaseEntity {
+    @Id
+    private String id;
+    @Lob
+    private Clob dto;
+    @With
```
[the remark this answers is not in the document]
Comment (lsjolinder) [reply src/main/java/se/sundsvall/casemanagement/integration/db/model/CaseEntity.java:29]: @sonatype-lift ignore

File: src/main/java/se/sundsvall/casemanagement/integration/db/model/CaseEntity.java [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+import lombok.Builder;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.ToString;
+import lombok.With;
+
+@Entity
+@Getter
+@ToString
+@Builder(setterPrefix = "with")
```
[the remark this answers is not in the document]
Comment (lsjolinder) [reply src/main/java/se/sundsvall/casemanagement/integration/db/model/CaseEntity.java:21]: @sonatype-lift ignore

File: src/main/java/se/sundsvall/casemanagement/configuration/RetryProperties.java [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+import lombok.Getter;
+import lombok.Setter;
+
+@Getter
+@ConfigurationProperties(prefix = "retry")
+public class RetryProperties {
+
+    private final int maxAttempts = 3;
+
+    private final Duration initialDelay = Duration.ofSeconds(2);
```
[the remark this answers is not in the document]
Comment (lsjolinder) [reply src/main/java/se/sundsvall/casemanagement/configuration/RetryProperties.java:16]: @sonatype-lift ignore

**Pull request merged**